VirtualH89/Src/AddressBus.cpp

```diff
@@ -10,10 +10,11 @@
 #include "MemoryDecoder.h"
 #include "InterruptController.h"
 
-AddressBus::AddressBus(InterruptController* ic): ic_m(ic), mem(NULL)
+AddressBus::AddressBus(InterruptController* ic):
```

Comment:
As an example, here are two versions of the same file. before and after 
running uncrustify with the config file in VirtualH89. Notice the 
function definitions and initializers, I don't think that is your 
intended format. So, that uncrust.cfg file does not seem to be doing a 
complete job of formatting the way you want.

On 03/15/2016 08:32 PM, Mark Garlanger wrote:
>
> In VirtualH89/Src/AddressBus.cpp 
> <https://github.com/mgarlanger/VirtualH89/pull/17#discussion_r56270222>:
>
> > @@ -10,10 +10,11 @@
> >  #include "MemoryDecoder.h"
> >  #include "InterruptController.h"
> >
> > -AddressBus::AddressBus(InterruptController* ic): ic_m(ic), mem(NULL)
> > +AddressBus::AddressBus(InterruptController* ic):
>
> Did uncrustify make these changes to AddressBus, without any other 
> changes made to the file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/mgarlanger/VirtualH89/pull/17/files/5775837f9d9ba3d8fca88f1b5d2c5b38aa4f982d#r56270222>
>